submissions/2021-11-08-QMDataset-pubchem-set1-single-points/README.md

```diff
@@ -0,0 +1,50 @@
+## Description
+
+This is a single point energy calculation of PubChem set 1 (1-2500) molecules. Detailed description on how the data is generated can be found at https://github.com/openmm/qmdataset/tree/main/pubchem.
+
+## General Information
+
+ - Date: 2021.11.08
+ - Class: Basic dataset 
+ - Purpose: Energy calculation
+ - Collection: BasicDataset
+ - Name: SPICE PubChem Set 1 Single Points Dataset v1.0
+ - Number of unique molecules:        2372
+ - Number of filtered molecules:      0
+ - Number of conformers:              118606
+ - Number of conformers min mean max: 6  50.00 100
+ - Set of charges: [-8.0, -4.0, -3.0, -2.0, -1.0, 0.0, 1.0, 2.0]
+ - Dataset Submitter: Josh Horton, Pavan Behara, David Dotson
+ - Dataset Generator: Peter Eastman
+ - Dataset Source: https://github.com/openmm/qmdataset/tree/main/pubchem
+
+## Changelog
+
+Here any information regarding dataset changes are recorded.
+
+## QCSubmit generation pipeline
+
+ - `Dataset_Generation.ipynb`: This notebook shows how the dataset was prepared from the input files. 
+ 
+## QCSubmit Manifest
+
+- `Dataset_Generation.ipynb`: Dataset creation notebook with instructions for submission.
+- `dataset.json.bz2`: The compressed constrained optimization dataset ready for submission.
```

Comment:
These just need cleaning up a bit, they must have been copied from the last readme.